nuxeo-core/nuxeo-core-mongodb/src/test/java/org/nuxeo/ecm/core/mongodb/TestMongoDBIndices.java

```diff
@@ -83,11 +84,13 @@ public void shouldFailWhenCreatingExistingChildNameDocument() {
             fail("should throw a ConcurrentUpdateException");
         } catch (ConcurrentUpdateException cue) {
             assertEquals(SC_CONFLICT, cue.getStatusCode());
+            String mongoDBName = Framework.getProperties().getProperty(MONGODB_DBNAME_PROPERTY);
```

Comment:
```suggestion
            String mongoDBName = Framework.getProperty(MONGODB_DBNAME_PROPERTY);
```
no?

Reply:
yep i just copied/past the code that was on `MongoDBFeature`

Reply:
to keep the things simple/more readable i took https://github.com/nuxeo/nuxeo/pull/3675#discussion_r367923480 into account which removes the dependency to `Framework.getProp...`